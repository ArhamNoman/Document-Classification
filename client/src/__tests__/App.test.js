import {App} from '../App';
import {shallow} from 'enzyme'; 
import React from 'react'; 
import toJson from 'enzyme-to-json'; 

const component = shallow(<App />);

describe('Login Component', () => {
    it('renders without crashing given the required props', () => {
        const props = {}
        const wrapper = shallow(<App {...props} />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
    
    it('Login provides two login options', () => {
        const wrapper = component.find('ProtectedRoute');
        expect(wrapper.length).toBe(2);
        // console.log(component.debug())

    })
})
