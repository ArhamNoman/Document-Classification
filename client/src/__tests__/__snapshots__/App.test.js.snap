// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing given the required props 1`] = `
<div>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <ProtectedRoute
      component={[Function]}
      path="/admin"
    />
    <ProtectedRoute
      component={[Function]}
      path="/faculty"
    />
    <Route
      component={[Function]}
      path="*"
    />
  </Switch>
</div>
`;
